### PR TITLE
Feature/dwr 439 item info

### PR DESCRIPTION
**Pull request by bgsavage**
Added Calculator and Interpretive Guide (configurable) to item info tab.

### CODE REVIEW

File: webapp/src/main/resources/application.yml [hunk 1]
```diff
@@ -84,4 +84,7 @@ app:
     url: http://wkhtmltopdf/
     options:
       dpi: 276
-      page-size: Letter
\ No newline at end of file
+      page-size: Letter
+
+  external-links:
+    interpretive-guide: http://www.smarterbalanced.org/educators/
```
Comment (bgsavage) [inline webapp/src/main/resources/application.yml]: This is a dummy value for now until we have an actual interpretive guide to point to.

File: webapp/src/main/webapp/src/app/app.component.html [hunk 2]
```diff
@@ -18,7 +18,7 @@
           <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
             <ul class="nav navbar-nav navbar-right">
               <li class="navbar-link"><a href="#">{{'labels.navigation.user-guide' | translate}}</a></li>
-              <li class="navbar-link"><a href="#">{{'labels.navigation.interpretive-guide' | translate}}</a></li>
+              <li class="navbar-link"><a href="{{interpretiveGuide}}" target="_blank">{{'labels.navigation.interpretive-guide' | translate}}</a></li>
```
Comment (bgsavage) [inline webapp/src/main/webapp/src/app/app.component.html]: Updated the link on the top nav section as well.

File: webapp/src/main/webapp/src/app/assessments/items/item-viewer/item-viewer.component.spec.ts [hunk 3]
```diff
@@ -29,8 +30,3 @@ describe('ItemViewerComponent', () => {
   });
 });
 
-class MockUserService {
-  getCurrentUser() {
-    return Observable.of(new User());
-  }
-}
```
Comment (bgsavage) [inline webapp/src/main/webapp/src/app/assessments/items/item-viewer/item-viewer.component.spec.ts]: moved to shareable location.

File: webapp/src/main/java/org/opentestsystem/rdw/reporting/search/user/AppSettings.java [hunk 4]
```diff
@@ -5,6 +5,7 @@
     private String irisVendorId;
     private String irisUrl;
     private String analyticsTrackingId;
+    private String interpretiveGuide;
```
Comment (malaffoon) [inline webapp/src/main/java/org/opentestsystem/rdw/reporting/search/user/AppSettings.java]: nit: should this be interpretiveGuideUrl?

Comment (bgsavage) [inline webapp/src/main/java/org/opentestsystem/rdw/reporting/search/user/AppSettings.java]: Yep, I'll update this in a future pr

**Pull request merged**